common/tests/test_helpers.py

```diff
@@ -9,12 +9,12 @@
 
 
 class CommonHelperTests(TestCase):
-    def test_prerender_urls(self):
+    def test_is_sitemap_urls(self):
         urls = SitemapPages()
         for url in urls:
             self.assertTrue(is_sitemap_url(url), 'Should be able to prerender ' + url)
```

Comment:
Should update test messages too.

Reply:
Just finished removing those from both files.